plexapi/media.py

```diff
@@ -293,21 +307,14 @@ class SubtitleStream(MediaPartStream):
         Attributes:
             TAG (str): 'Stream'
             STREAMTYPE (int): 3
-            forced (bool): True if this is a forced subtitle
-            format (str): Subtitle format (ex: srt).
-            key (str): Key of this subtitle stream (ex: /library/streams/212284).
-            title (str): Title of this subtitle stream.
     """
     TAG = 'Stream'
     STREAMTYPE = 3
 
     def _loadData(self, data):
         """ Load attribute values from Plex XML response. """
         super(SubtitleStream, self)._loadData(data)
-        self.forced = cast(bool, data.attrib.get('forced', '0'))
-        self.format = data.attrib.get('format')
-        self.key = data.attrib.get('key')
-        self.title = data.attrib.get('title')
```

Comment:
These attributes are still valid for subtitle streams.

Reply:
key and title are inherit. As for forced and format, maybe I thought they were inherit as well or my sample xmls didn't show them.

Reply:
Restore doc string for `forced` and `format`.